.github/workflows/ci.yml

```
@@ -26,6 +26,7 @@ env:
 jobs:
   setup:
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)